src/index.ts

```diff
@@ -438,6 +443,9 @@ export class GossipSub extends EventEmitter<GossipsubEvents> implements Initiali
     this.score = new PeerScore(this.opts.scoreParams, this.metrics, {
       scoreCacheValidityMs: opts.heartbeatInterval
     })
+
+    this.maxInboundStreams = options.maxInboundStreams ?? 32
+    this.maxOutboundStreams = options.maxOutboundStreams ?? 128
```

Comment:
```suggestion
    this.maxInboundStreams = options.maxInboundStreams
    this.maxOutboundStreams = options.maxOutboundStreams
```
maybe we just have no default (and just fall back to the upstream default)